database.py

```diff
@@ -20,7 +23,7 @@ def sql_string(object):
     return "\'{}\'".format(str(object))
 
 
-def update_message_in_db(message: discord.Message):
+def update_message_in_db(message: discord.Message, guild_id: int):
```

Comment:
You can get guild_id from message.guild.id:

[https://discordpy.readthedocs.io/en/latest/api.html?discord.CategoryChannel#discord.Message.guild](https://discordpy.readthedocs.io/en/latest/api.html?discord.CategoryChannel#discord.Message.guild)

Reply:
well damn